From 335e238a611c3c9796e14e1bff968ea519703f7c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Petr=20P=C3=ADsa=C5=99?= <ppisar@redhat.com>
Date: Tue, 10 Aug 2010 14:01:55 +0200
Subject: [PATCH] Move AuErrorDB to SHAREDIR

AuErrorDB list of error messages. Non-executables should not reside in LIBDIR.
---
 README               |    2 +-
 lib/audio/AuErrDes.c |    2 +-
 lib/audio/Imakefile  |    6 +++---
 3 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/README b/README
index 790bc58..0d852c0 100644
--- a/README
+++ b/README
@@ -175,7 +175,7 @@ A complete installation should include the following files:
 	/usr/man/man3/*.3			API man pages
 
 	/usr/lib/libaudio.[a|so]		API library
-	/usr/lib/X11/AuErrorDB			Error database
+	/usr/share/X11/AuErrorDB		Error database
 
 	/usr/bin/X11/nasd			NAS server for your
 						hardware (voxware,
diff --git a/lib/audio/AuErrDes.c b/lib/audio/AuErrDes.c
index 157423a..f5e8ea7 100644
--- a/lib/audio/AuErrDes.c
+++ b/lib/audio/AuErrDes.c
@@ -23,7 +23,7 @@
  */
 
 #ifndef ERRORDB
-#define ERRORDB "/usr/lib/X11/AuErrorDB"
+#define ERRORDB "/usr/share/X11/AuErrorDB"
 #endif /* !ERRORDB */
 
 #include "Alibint.h"
diff --git a/lib/audio/Imakefile b/lib/audio/Imakefile
index db73732..2f2e740 100644
--- a/lib/audio/Imakefile
+++ b/lib/audio/Imakefile
@@ -79,7 +79,7 @@ XCOMM JET - see if we should avoid streams
     COMPAT_OBJS = AuGetBest.o
 #endif /* ProjectX < 5 */
 
-     EDB_DEFINES = -DERRORDB=\"$(LIBDIR)/AuErrorDB\"
+     EDB_DEFINES = -DERRORDB=\"$(SHAREDIR)/AuErrorDB\"
 
         DEFINES = $(MALLOC_DEFINES) $(SYSV_DEFINES)
    CONN_DEFINES = ConnectionFlags
@@ -137,11 +137,11 @@ SpecialObjectRule(globals.o,/**/,$(CONN_DEFINES))
 BuildIncludes($(HEADERS),audio,..)
 InstallMultiple($(HEADERS),$(INCDIR))
 #if ProjectX >= 5
-InstallNonExecFile(AuErrorDB,$(LIBDIR))
+InstallNonExecFile(AuErrorDB,$(SHAREDIR))
 #else
 install::
 	$(MKDIRHIER) $(LIBDIR)
-InstallNonExec(AuErrorDB,$(LIBDIR))
+InstallNonExec(AuErrorDB,$(SHAREDIR))
 #endif
 
 #if defined(HPArchitecture) && !defined(LinuxArchitecture) && !defined(GNUArchitecture) && !defined(DarwinArchitecture)
-- 
1.7.6.1

